### PR TITLE
Feature/Comments comments

### PR DESCRIPTION
**Pull request by TinyRogue**

### CODE REVIEW

File: ArithmeticsDiff.java [hunk 1]
```diff
@@ -1,5 +1,15 @@
 public class ArithmeticsDiff implements IArithmeticsDiff{
     public double difference(double A, double B){
-        return A-B;
+        return A-B; //Tell me why! No space is here na na na
     }
 }
+
+/*
+    Tell me why, tell me why, tell me why
```
Comment (Roller23) [inline ArithmeticsDiff.java:8]: Beautiful, I cried

File: ArithmeticsDiv.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 public class ArithmeticsDiv implements IArithmeticsDiv {
-
+    //Beware of division by 0!
```
Comment (Roller23) [inline ArithmeticsDiv.java:2]: Good catch

File: ArithmeticsDiv.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
     @Override
     public double division(double A, double B) {
         return A / B;
```
Comment (Roller23) [inline ArithmeticsDiv.java:5]: Got to fix this

**Review by Roller23 — CHANGES_REQUESTED** [review]
There's a vulnerabilitie to fix

**Comment by TinyRogue** [comment]
Wiktor, what have You done?!

**Pull request closed without merging**